guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

```diff
@@ -2,8 +2,9 @@
 
 = Cloning {ProjectServer}
 
-When you upgrade {ProjectServer}, you can optionally create and upgrade a clone of your {Project} to ensure that you do not lose any data while you upgrade.
-After your upgrade is complete, you can then decommission the earlier version of {ProjectServer}.
+You can clone the {ProjectServer} to create  instances for trying out upgrades and the migration of instances to a different machine or operating system.
+This is an optional step to provide more flexibility during the upgrade.
+Note: The created instances are not supposed to run in parallel in a production environment, please decommission any testing instances after completing the tests or any old instances after completing the migration.
```

Comment:
```suggestion
Note: The created instances are not supposed to run in parallel in a production environment. You must decommission any testing instances after completing the tests or any old instances after completing the migration.
```
Agree to Adam's view + We do not use polite terms.

Reply:
Please stick to one sentence per line.